devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -415,6 +446,18 @@ private boolean isSomfySituo4() {
 	isSomfy() && device.getDataValue("model") == "Situo 4 Zigbee"
 }
 
+private boolean isEWeLink() {
+	device.getDataValue("manufacturer") == "eWeLink"
+}
+
+private boolean isEWeLinkWb01() {
+	isEWeLink() && device.getDataValue("model") == "WB01"
+}
+
+private boolean isEWeLinkSnzb01p() {
+	isEWeLink() && device.getDataValue("model") == "SNZB-01P"
+}
+
```

Comment:
Couldn't we just get away with `isEWeLink()` and removing model checks, as there's no any device specific configuration and only some exceptions for all manufacturer's devices?